quickstarts/WFLY-19335-kubernetes-readmes-for-quickstarts.adoc

```diff
@@ -0,0 +1,166 @@
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY[WFLY-19335]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/WFLY[WFLY-19334 - Add PR testing of Quickstarts on Kubernetes]
+* https://issues.redhat.com/browse/WFLY-19337[WFLY-19337 - Enable the ejb-txn-remote-call quickstart on Kubernetes CI testing] - at the time of writing this quickstart is not working yet on the Kubernetes CI.
+* At the time of writing these quickstarts are not working yet on the Kubernetes CI:
+** https://issues.redhat.com/browse/WFLY-19336[WFLY-19336 - Enable the ejb-txn-remote-call quickstart on Kubernetes CI testing ]
+** https://issues.redhat.com/browse/WFLY-19337[WFLY-19337 - Enable the ejb-txn-remote-call quickstart on Kubernetes CI testing]
+
+=== Stability Level
+// Choose the planned stability level for the proposed functionality
+* [ ] Experimental
+
+* [ ] Preview
+
+* [x] Community
+
+* [ ] default
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+Just the Quickstarts
+
+=== Other Interested Projects
+
+None that I am aware of
+
+=== Relevant Installation Types
+// Remove the x next to the relevant field if the feature in question is not relevant
+// to that kind of WildFly installation
+* [x] Traditional standalone server (unzipped or provisioned by Galleon)
+
+* [ ] Managed domain
+
+* [ ] OpenShift s2i
+
+* [x] Bootable jar
+
+The choices here aren't really relevant since for Kubernetes we don't rely on OpenShift CI. Instead we provide an image that contains the built server. For one set of the quickstarts this is a 'traditional' server provisioned using WildFly Glow and the WildFly Maven Plugin, and for the other set of quickstarts the server is a bootable jar, again provisioned using WildFly Glow and the WildFly Maven Plugin.
+
+== Requirements
+
+=== Hard Requirements
+
+* The quickstarts mentioned in the Overview section must:
+** Have instructions in the README for how to deploy and test the application using Helm on Kubernetes
+** Have tests runnable on GitHub Actions that perform the same steps as the instructions in the README
+* The instructions to deploy on Kubernetes are overall quite similar to how to deploy on OpenShift, except:
+** `kubectl` is used instead of `oc` when interacting with Kubernetes
+*** There is no need to log in to Kubernetes, so there is no `kubectl login` equivalent of `oc login`
+** Some of the quickstarts require installing things to Kubernetes beforehand
+*** When these prerequisites are different in Kubernetes to OpenShift, the README should reflect that
+** Kubernetes does not have s2i, instead we build the image using `mvn wildfly:image`, and tag and deploy that to the Kubernetes registry
```

Comment:
This add a requirement to have `podman` or `docker` installed as `mvn wildly:image` will delegate to these tools to build the images